pom.xml

```diff
@@ -34,7 +34,8 @@
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     <maven.build.timestamp.format>yyyyMMddHHmm</maven.build.timestamp.format>
     <appengine.api.version>1.9.40</appengine.api.version>
-    <docker.label.prefix>async-</docker.label.prefix>
+    <docker.tag>8-jre</docker.tag>
```

Comment:
Sure. Here's the issue: https://github.com/GoogleCloudPlatform/openjdk-runtime/issues/8